src/common/transformations/include/transformations/common_optimizations/rms_fusion.hpp

```diff
@@ -5,15 +5,24 @@
 #pragma once
 
```

Comment:
[random spot]
@jane-intel, @dmitry-gorokhov What do you think about adding reference impls for common internal ops? Does it make sense? Should it become a common practice in the future?

Technically, plugins can test the operation by defining decomposed pattern, so the reference flow will use reference impls of public ops. But if we want to have more isolated tests in the future or more clear spec of internal ops, then ref impl may help. So please share your thoughts.

Reply:
Hello @vladimir-paramuzov, there are existing reference implementations and specs for common internal ops like AUGRUCell/AUGRUSequence. So I've created a PR with reference implementation and spec for common internal::RMS as well (still there is more flexibility to provide further modifications and improvements of such internal operations):
- https://github.com/openvinotoolkit/openvino/pull/24556
- https://github.com/openvinotoolkit/openvino/pull/24564